docs/how-to-guides/themes/theme-json.md

```diff
@@ -169,6 +169,9 @@ The settings section has the following structure:
 {
 	"version": 1,
 	"settings": {
+		"border": {
+			"customRadius": false
```

Comment:
This landed in WordPress 5.8.

Reply:
It was 5.8.1 - https://github.com/WordPress/wordpress-develop/pull/1513.

Reply:
Oh, thanks for catching that. I see what happened. I think it's fine to leave it in the "WordPress" as it was a bug and should have been ported to 5.8 anyway.

However, it prompts us to improve the docs to better pin a given setting/style to a given WordPress release (we'll certainly need this for WordPress 5.9).

Reply:
The handbook setup is very limited, so I wonder if we should just have different subsections per release. Something along the lines of:

- Settings
  - WordPress 5.8
  - WordPress 5.9
  - Gutenberg

Reply:
> However, it prompts us to improve the docs to better pin a given setting/style to a given WordPress release (we'll certainly need this for WordPress 5.9).

💯  This would be useful in docs.

Reply:
@mkaz, do you have any suggestions for this?

Reply:
For docs, I think we just need to use `@since` comments similar to how we would in code. It's not quite valid JSON, but if we consider the file to be JSONC (json with comments) we could do:

```jsonc
"settings": {
		"border": {
			// @since wp5.8.1
			"customRadius": false
			...
```